dev-embedded/u-boot-tools/u-boot-tools-2020.10.ebuild

```diff
@@ -12,59 +12,71 @@ SRC_URI="ftp://ftp.denx.de/pub/u-boot/${MY_P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
-IUSE=""
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="minimal"
 
 BDEPEND="
 	sys-devel/bison
-	sys-devel/flex
-"
+	sys-devel/flex"
 
-S=${WORKDIR}/${MY_P}
+S="${WORKDIR}"/${MY_P}
 
 src_prepare() {
 	default
 	sed -i 's:\bpkg-config\b:${PKG_CONFIG}:g' \
-		scripts/kconfig/lxdialog/check-lxdialog.sh \
 		scripts/kconfig/Makefile \
 		tools/Makefile || die
 }
 
 src_configure() {
-	tc-export CC PKG_CONFIG
+	tc-export AR CC PKG_CONFIG
 }
 
-src_compile() {
-	# Unset a few KBUILD variables. Bug #540476
-	unset KBUILD_OUTPUT KBUILD_SRC
-
+_emake() {
 	emake \
 		V=1 \
+		NO_SDL=1 \
+		HOSTSTRIP=: \
+		STRIP=: \
+		AR="${AR}" \
 		CC="${CC}" \
 		HOSTCC="${CC}" \
 		HOSTCFLAGS="${CFLAGS} ${CPPFLAGS}"' $(HOSTCPPFLAGS)' \
 		HOSTLDFLAGS="${LDFLAGS}" \
-		tools-only_defconfig
+		CONFIG_ENV_OVERWRITE=y \
+		"$@"
+}
+
+src_compile() {
+	# Unset a few KBUILD variables. Bug #540476
+	unset KBUILD_OUTPUT KBUILD_SRC
 
 	emake \
 		V=1 \
-		NO_SDL=1 \
-		HOSTSTRIP=: \
-		STRIP=: \
+		AR="${AR}" \
 		CC="${CC}" \
 		HOSTCC="${CC}" \
 		HOSTCFLAGS="${CFLAGS} ${CPPFLAGS}"' $(HOSTCPPFLAGS)' \
 		HOSTLDFLAGS="${LDFLAGS}" \
-		CONFIG_ENV_OVERWRITE=y \
-		tools-all
+		tools-only_defconfig
+
+	if use minimal; then
+		_emake envtools
+	else
+		_emake tools-all
+	fi
 }
 
 src_test() { :; }
 
 src_install() {
 	cd tools || die
-	dobin bmp_logo dumpimage fdtgrep gen_eth_addr img2srec mkenvimage mkimage
 	dobin env/fw_printenv
+
+	if ! use minimal; then
+		dobin bmp_logo dumpimage fdtgrep gen_eth_addr img2srec mkenvimage mkimage
+	fi
+
 	dosym fw_printenv /usr/bin/fw_setenv
 	insinto /etc
 	doins env/fw_env.config
```

Comment:
Why not wrap the whole string passed to doman in quotes?

Reply:
I prefer to quote only `${S}`, looks nicer